### DISCUSSION TITLE
I am getting Permission denied when trying to open a PR

### DISCUSSION THREAD

**Discussion by burakaktasfe** [serverless/serverless#10287] [General]
Hi guys,

I need to update some jackson dependencies from plugins which is causing a security violation. Almost same like; https://github.com/serverless/serverless/pull/5653. However, I am not able to publish a PR and I am getting permission denied error;

```
➜  serverless git:(update-jackson-dependencies) ✗ git push -u origin update-jackson-dependencies
ERROR: Permission to serverless/serverless.git denied to burakaktasfe.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists
```

Can you please guide me if there are any steps to follow?

Thanks,
Burak

**Comment by pgrzesik** [comment]
Hello @burakaktasfe - if you'd like to submit a PR, you need to first create your own fork of `serverless` repository and then submit a PR from the fork. It's complaining about permission denied because you're trying to push directly to `serverless/serverless` repo which is not permitted. 

Please let me know if you need any additional assistance, I'll be happy to help.

**Reply by burakaktasfe** [reply]
Thanks @pgrzesik ! I will do.

**Reply by burakaktasfe** [reply]
I created the PR - https://github.com/serverless/serverless/pull/10289
